sql/transactions.sql

```diff
@@ -0,0 +1,185 @@
+-- Simple SQL script to concatenate EQR transactions and apply real dtypes
+-- However, the ENUMs for low-cardinality strings are not reducing the file size.
+-- Not sure if they're actually being dictionary encoded in the Parquet output.
+-- Ideally we would identify the universe of values dynamically rather than manually.
+
+-- To run the conversion, from the top level of the repo run:
+-- duckdb < sql/transactions.sql
+
+-- This has only been tested on the 2022 Q4 data.
+-- It takes less than 1 minute to run on 200M rows.
```

Comment:
I was able to grab 5 random quarters and extract them to parquet with dagster, then concatenate them all into the same schema using this query without any errors. Somehow I'm not getting much of a size improvement by turning the low-cardinality strings into ENUMs. Not sure what I'm doing wrong.  For 5 quarters of data I ended up with a 12GB file, and it took less than 2min to create. Maybe the original Parquet outputs are already identifying the low-cardinality `VARCHAR` columns and using dictionary encoding even without the ENUM?